src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveforbarn/GjeldendeBarnRepository.kt

```diff
@@ -14,12 +14,12 @@ interface GjeldendeBarnRepository : RepositoryInterface<BarnTilUtplukkForOppgave
 
     // language=PostgreSQL
     @Query("""
-        SELECT b.id behandling_id, s.fodselsnummer fodselsnummer_soker, b2.fodselsnummer fodselsnummer_barn, b2.fodsel_termindato 
+        SELECT b.id behandling_id, pi.ident fodselsnummer_soker, bb.person_ident fodselsnummer_barn, bb.fodsel_termindato termindato_barn
         FROM gjeldende_iverksatte_behandlinger b
-            JOIN soknad_grunnlag gs ON gs.behandling_id = b.id
-            JOIN soknadsskjema s ON gs.soknadsskjema_id = s.id
-            JOIN soknad_barn b2 ON s.id = b2.soknadsskjema_id
-        WHERE  b.stonadstype=:stønadstype AND EXISTS(SELECT 1 FROM andel_tilkjent_ytelse aty
+         JOIN fagsak fs ON fs.id = b.fagsak_id
+         JOIN (SELECT DISTINCT ON(pi.fagsak_person_id) * FROM person_ident pi ORDER BY pi.fagsak_person_id, pi.opprettet_tid DESC) pi ON pi.fagsak_person_id = fs.fagsak_person_id
```

Comment:
Denne var ny for meg 👍 . Tenker vi burde legge til et aktivtflagg på person_ident en gang :)